ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -314,7 +315,20 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
   },
 ]
 
-const RNA_SEQ_DETAIL_FIELDS = ['zScore', 'pValue', 'pAdjust']
+const RNA_SEQ_COLUMNS = [
+  { name: 'individual', content: '', width: 3 },
+  { name: 'zScore', content: 'Z-Score', width: 3, format: ({ zScore }) => (zScore ? zScore.toPrecision(3) : null) },
+  { name: 'pValue', content: 'P-Value', width: 3, format: ({ pValue }) => (pValue ? pValue.toPrecision(3) : null) },
+  { name: 'pAdjust', content: 'P-Adjust', width: 3, format: ({ pAdjust }) => (pAdjust ? pAdjust.toPrecision(3) : null) },
```

Comment:
`DataTable` will set the column widths equally by default if you don't specify a width so unless you want different sizes you should leave the width prop out